src/engine/game/map_settings.ts

```diff
@@ -42,4 +42,5 @@ export interface MapSettings {
   readonly rotation?: Rotation;
 
   getOverrides(): Array<SimpleConstructor<unknown>>;
-}
\ No newline at end of file
+  getRiversLayer?(): ReactNode
```

Comment:
Instead of a getRiversLayer, should we just have a textures layer instead? This will give us the option to add flavor to certain hexes (like a cool mountain range or something).

Reply:
Yea, that's something I was thinking about, too. I'll rename it to "getTexturesLayer"